### DISCUSSION TITLE
Request: Restore playhead position (and possibly viewer size/position?) when re-opening clip

### DISCUSSION THREAD

**Discussion by adamwilt** [videovillage/community#131] [Ideas]
Screen could restore playhead position, and possibly viewer size and position, when re-opening a clip.

TL;DR version: Restoring playhead position would be very useful; please add this (low priority, but it would be helpful). Restoring viewer size/position would be handy some times, but it might be annoying other times; please read the blatheration below and see if it makes any sense to you (much lower priority; possibly not worth doing at all!).

Restoring playhead position would be very helpful. I often park the playhead at a point of interest, and would like to see that point of interest restored when a clip is re-opened. (And yes, I can work around that by marking an In or Out point before closing the clip, so it's not like the lack of playhead restoration is preventing me from profitably using Screen.)

Restoring viewer size/position might be handy, too, but it's also problematic, so I'm not sure about this. Automatically restoring size/position interferes with the current "open all viewers at ~80% of window size, centered" operation. And it's less annoying (at least for me) to re-drag viewers into position than having to re-navigate to a particular point of interest on the timeline.

There are often times where I have two or three clips up at the same time so I can compare them, and I tend to rearrange 'em top-to-bottom or side-by-side. In many cases it would be nice if, when I re-open these clips, they come back to their previously set positions onscreen. But there are just as many time where I re-open a clip *not* in the context of doing an A/B comparison, and restoring the previous size/position adds no value. If it's a clip I haven't been using recently — it's a clip from an old project, for example — having it come up in an unexpected size/position is arguably *worse* than the "80%, centered" default. Thus I'm ambivalent about this request even for my own use cases, and I don't know if this is the sort of thing other users would want — or would actively dislike. It's one of those things where I first thought, "gee, I wish Screen did this", but on further reflection, I'm not so sure it's a good idea for typical workflows. And yes, there could be a Preferences setting to enable/disable viewer restoration. But again, is this just adding more complication to slightly improve an atypical workflow?
